omines/antispam-bundle/0.1/config/packages/antispam.yaml

```diff
@@ -0,0 +1,38 @@
+#
+# This sample configuration sets up a default anti-spam profile that will already stop a lot of
+# form spam with minimal effort and none to minimal user inconvenience.
+#
+# To get started right away read the Quickstart at https://omines.github.io/antispam-bundle/quickstart/
+#
+# For more details on the options available visit https://omines.github.io/antispam-bundle/configuration/
+#
+antispam:
+    profiles:
+        default:
+            # Insert a honeypot called "email_address" on all forms to lure bots into filling it in
+            honeypot: email_address
+
+            # Reject all forms that have been submitted either within 3 seconds, or after more than an hour
+            timer:
+                min: 3
+                max: 3600
+
+            #
+            # The measures above should already have notable effect on the amount of spam that gets through
+            # your forms. Still getting annoying amounts? Analyze the patterns of uncaught spam, then
+            # consider uncommenting and modifying some of the examples below after careful consideration
+            # about their impact.
+            #
+
+            # Reject text fields that contain (lame attempts at) HTML or BBCode
+#            banned_markup: true
+
+            # Reject text fields that consist for more than 40% of Cyrillic (Russian) characters
+#            banned_scripts:
+#                scripts: [ cyrillic ]
+#                max_percentage: 40
+
+            # Reject fields that contain more than 3 URLs, or repeat a single URL more than once
+#            url_count:
+#                max: 3
+#                max_identical: 1
```

Comment:
```suggestion
#                max_identical: 1

when@test:
    antispam:
        # disable in tests
        passive: true
```

I know passive mode via config is not yet implemented but what about adding this for when it is?

Reply:
That is actually a brilliant idea. Thnx!